src/index.html

```diff
@@ -5,9 +5,346 @@
+          <div class="testimonials__placing">
+            <img
+              src="../src/images/people/person-1.png"
+              alt="McKinney-photo"
+              class="testimonials__photo"
+            >
+            <h3 class="testimonials__name">
+              Leslie Mckinney
+            </h3>
+            <h4 class="testimonials__position">
+              CEO
+            </h4>
+            <p class="testimonials__description">
+              Nice work, Certificates National Association of Realtors (USA)
+            </p>
+          </div>
+
+          <div class="testimonials__placing">
+            <img
+            src="../src/images/people/person-2.png"
+            alt="Hawkins-photo"
+            class="testimonials__photo"
+            >
+            <h3 class="testimonials__name">
+              Scarlett Hawkins
+            </h3>
+            <h4 class="testimonials__position">
+              Senior Vice President
+            </h4>
+            <p class="testimonials__description">
+              My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart
+            </p>
+          </div>
+
+          <div class="testimonials__placing">
+            <img
+            src="../src/images/people/person-3.png"
+            alt="Henry-photo"
+            class="testimonials__photo"
+            >
+            <h3 class="testimonials__name">
+              Max Henry
+            </h3>
+            <h4 class="testimonials__position">
+              Chief Design Officer
+            </h4>
+            <p class="testimonials__description">
+              When my lovely valley of steam rises and half-day sun is above an impermeable
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section class="contacts main__contacts" id="contacts">
+        <h2 class="heading contacts__heading">
+          Contact us
+          <div class="micro-logo heading__micro-logo"></div>
+        </h2>
+        <div class="contacts__wrapper">
+          <div class="connection contacts__connection">
+            <h3 class="phone-text connection__phone-text">Phone</h3>
+            <a
+              href="+123455555555"
+              class="phone-number connection__phone-number"
+            >
+              +1 234 555-55-55
+            </a>
+            <h3 class="email-text connection__email-text">Email</h3>
+            <a
+              href="mailto:hello@miami.com"
+              class="email connection__email"
+            >
+              hello@miami.com
+            </a>
+          </div>
+          <div class="address contacts__address">
+            <h3 class="address-text address__address-text">Address</h3>
+            <h4 class="destination address__destination">
+              400 first ave. n.<br> suite 700<br>Minneapolis, MN<br> 55401
+            </h4>
+          </div>
+          <form
+            action="mate-academy.com"
+            method="POST"
```

Comment:
```suggestion
          <form
            action="#"
            method="POST"
```
Let's fix this and that will remove the error that we have now when submitting your form:
![1](https://user-images.githubusercontent.com/66570142/114711716-f68b0480-9d37-11eb-98f1-933ea35ba669.png)